test.c

```diff
@@ -251,12 +252,14 @@ int main()
         RETURN_ERROR;
     }
     else
-        printf("SUCCESS\tGot symbol from global handle: %p\n", fwrite_local);
-    char * hello_world = "Hello world from local fwrite!\n";
+        printf("SUCCESS\tGot symbol from global handle: %p\n", *(void **)&fwrite_local);
+    {
+    const char *hello_world = "Hello world from local fwrite!\n";
     fwrite_local(hello_world,sizeof(char),strlen(hello_world),stderr);
     fflush(stderr);
+    }

```

Comment:
I am not sure what convention we follow in the rest of the repo, but just to understand, in this case you did not indented intentionally as the scope is not associated with a `if`/`for`/`while`?

Reply:
Should I indent those 3 lines in that block by another level?

Reply:
This is the first scope of this kind in the codebase, but I briefly checked with https://zed0.co.uk/clang-format-configurator/ and all major styles indent lines inside scopes like this, so I think it make sense to indent those lines, thanks!

Reply:
I separated it in next commit (with explained commit message) and fixed it there.